modules/ocf_admin/manifests/create/redis.pp

```diff
@@ -40,14 +38,21 @@
     notify  => Service['redis-server'];
   }
 
-  ocf::repackage { 'stunnel4': backport_on => jessie } ->
-  augeas { '/etc/default/stunnel4':
-    lens    => 'Shellvars.lns',
-    incl    => '/etc/default/stunnel4',
-    changes =>  ['set ENABLED 1'],
-  } ~>
-  file { '/etc/stunnel/redis.conf':
-    content => template('ocf_admin/stunnel/redis.conf.erb'),
-  } ~>
-  service { 'stunnel4': }
+  package { 'hitch': }
+
+  # Yes, the underscore is intentional. We already have a user named "hitch".
+  user { '_hitch':
+    groups  => 'ssl-cert',
+    notify  => Service['hitch'],
+    require => Package['hitch'],
+  }
+
+  $pem_file = "/etc/ssl/private/${::fqdn}.pem"
+  file { '/etc/hitch/hitch.conf':
+    content => template('ocf_admin/hitch.conf.erb'),
+    notify  => Service['hitch'],
+    require => Package['hitch'],
+  }
+
+  service { 'hitch': }
```

Comment:
Would it be of any concern that this doesn't depend on `Package['hitch']`?

Reply:
At the moment it already does because the preceding resources `notify` it. Unless someone deletes those resources, it should be fine.